src/lib/plugin.js

```diff
@@ -310,6 +313,11 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   * _send (transfer) {
+    const validAccount = typeof transfer.account === 'string'
+    const validAmount = typeof transfer.amount === 'string' && +transfer.amount > 0
```

Comment:
Seems odd to set variables for these rather than just checking in the `if` statement